server/routes/routerRequestHandlers/retrieveAllGoalsForPrisoner.test.ts

```diff
@@ -8,7 +8,7 @@ import { aValidGoal } from '../../testsupport/actionPlanTestDataBuilder'
 
 jest.mock('../../services/educationAndWorkPlanService')
 
-describe('ViewGoalsController', () => {
+describe('retrieveAllGoalsForPrisoner', () => {
```

Comment:
Just fixing up some minor copy/paste typos I found in this test class
(I copy/pasted the whole test class into my new one above, hence I found the issues)